CHANGELOG.md

```diff
@@ -3,6 +3,16 @@
 ## [Unreleased](https://github.com/NamelessMC/Nameless/compare/v2.2.0...develop)
 > [Milestone](https://github.com/NamelessMC/Nameless/milestone/23)
 
+## [2.2.2](https://github.com/NamelessMC/Nameless/compare/v2.2.1...v2.2.2) - 2025-05-11
+### Added
+- Add methods to add to Smarty security policy functions and modifiers, and access to smarty append [#3643](https://github.com/NamelessMC/Nameless/pull/3643)
+
+### Changed
+- Add Group Sync logs to Debug link [#3650](https://github.com/NamelessMC/Nameless/pull/3650)
+
+### Fixed
+- Fixed report event for report endpoint [#3648](https://github.com/NamelessMC/Nameless/pull/3648)
+
```

Comment:
Need to add "Fix being able to react to your own posts" now

Reply:
Done